src/read/cfi.rs

```diff
@@ -4224,7 +4232,7 @@ mod tests {
         let contents = section.get_contents().unwrap();
         let input = &mut EndianSlice::new(&contents, LittleEndian);
         assert_eq!(
-            CallFrameInstruction::parse(input),
+            CallFrameInstruction::parse(input, mem::size_of::<usize>() as u8),
```

Comment:
For most tests this address size doesn't matter, and I mildly prefer to set it as a constant (4 or 8) since that's simpler.